frontend/public/components/nav.jsx

```diff
@@ -122,9 +123,35 @@ const navSectionStateToProps = (state, {required}) => {
     flags, canRender,
     activeNamespace: state.UI.get('activeNamespace'),
     location: state.UI.get('location'),
+    promotedCrds: state.UI.get('consoleExtensions'),
   };
 };
 
+// class CrdNSLink extends NavLink {
+//   static isActive (props, resourcePath, activeNamespace) {
+//     const href = stripNS(formatNamespacedRouteForCrd(props.resource, activeNamespace));
+//     return matchesPath(resourcePath, href) || matchesModel(resourcePath, props.model);
+//   }
+//    get to () {
+//     const { resource, activeNamespace } = this.props;
+//     return formatNamespacedRouteForCrd(resource, activeNamespace);
+//   }
+// }
+// class CrdClusterLink extends NavLink {
+//   static isActive (props, resourcePath) {
+//     return resourcePath === props.resource || _.startsWith(resourcePath, `${props.resource}/`);
+//   }
+//    get to () {
+//     return `/k8s/cluster/${referenceForCRD(this.props.resource)}`;
+//   }
+// }
+
+const formatNamespacedRouteForCrd = (crdRef, activeNamespace=getActiveNamespace()) => {
```

Comment:
I don't think we need a special route for CRDs. We should be able to format the route like any other resource?